schemas/oic.oic-link-schema.json

```diff
@@ -134,6 +134,35 @@
               }
             }
           }
+        },
+        "tag-locn": {
+          "allOf": [
+            {
+              "$ref": "oic.types-schema.json#/definitions/locn-room-list"
+            },
+            {
+              "description": "Location of the Resource or Device"
+            }
+          ]
+
+        },
+        "tag-pos-desc": {
+          "oneOf": [
+            {
+              "$ref": "oic.types-schema.json#/definitions/pos-descriptions"
+            },
+            {
+              "type": "array",
+              "readOnly": true,
```

Comment:
Why is it read only when its an array but not read only when its an enum?  It seems that if you can change one, the other should be changeable as well.

Reply:
One is a relative position against a known point that is set by the Device Vendor that would be fairly nonsensical to be remotely changed.  The other is a text description that could be initially set by the Device Vendor but which I can envision being changed by a Client if there is some alternative supported by the enumeration that makes better semantic sense to the end user.